portfolio/src/main/webapp/index.html

```diff
@@ -2,22 +2,49 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ruchi's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>Ruchi Shah's Portfolio</h1>
-      <p class="portfolio_text">This is my personal portfolio. Scroll to learn more about me.</p>
-      <!--<p class="randomMsg_text">Click here to get a random message:</p>
-       <button onclick="addRandomGreeting()">Hello World!</button> -->
-      <div id="greeting-container"></div>
+      <h1>MY PORTFOLIO</h1>
+
+      <div class="master-title">
+        <span class="letter" data-letter="R">R</span>
+        <span class="letter" data-letter="U">U</span>
+        <span class="letter" data-letter="C">C</span>
+        <span class="letter" data-letter="H">H</span>
+        <span class="letter" data-letter="I">I</span>
+        <span class="letter" data-letter="S">S</span>
+        <span class="letter" data-letter="H">H</span>
+        <span class="letter" data-letter="A">A</span>
+        <span class="letter" data-letter="H">H</span>
+      </div>
+
+      <!-- <p class="portfolio_text">This is my personal portfolio. Scroll to learn more about me.</p> -->
+      <h2>
+        <center> <p class="typewrite" data-period="2000" data-type='[ "Hi, Nice to meet you.", "Welcome to my Webpage.", "Excited to show you my portfolio.", "Enjoy!" ]'> 
```

Comment:
consider not using the `<center>` tags. it's old tech that's not supported in html5.
consider using flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/